src/components/Table.jsx

```diff
@@ -461,39 +501,47 @@ export default function Table() {
       </span>
     );
   };
+
   const handleAssigneeSort = () => {
```

Comment:
Too much duplicated code.
Merge all of these methods into a single function call.

Reply:
addressed